### DISCUSSION TITLE
License clarification

### DISCUSSION THREAD

**Discussion by reubano** [PrefectHQ/prefect#3827] [General]
Hi. Can you please clarify the UI and server license? You say that they are open source, however, they use the `Prefect Community License` which is not [`OSI Approved`](https://opensource.org/licenses/alphabetical). While I understand the need to limit competition, there may be an actual open source license that accomplishes your goals. Otherwise, I feel it would be more accurate to say that the UI and Server are `source available` and that Prefect is `open core`. Thanks for creating such a great project!

**Comment by jlowin** [comment]
Hi @reubano,

Thanks for your thoughts. We are reassessing some of our community licenses, particularly with regard to clarifying our support for academic, open-source, and non-profit entities. This may result in the adoption of a more common open-source license, or we may elect to continue with a proprietary one that more accurately reflects the particular circumstances of our company. In either case, it is imperative to us that  almost all of our users are indeed granted a license that aligns with a set of principles like those in the OSI’s Open Source Definition. 

When it comes to describing our software, we use the term "open-source" in the way that is commonly understood by the vast majority of our users, both technical and non-technical. Though we deeply appreciate the value the OSI has brought to affirming a set of standard and unambiguous open-source licenses, we are not members of the OSI, we are not seeking OSI certification, and we do not require our users to have any knowledge of the OSI. Therefore, to refer to our software as anything other than "open-source" would cause great confusion among a population of users that may not know or understand the distinction you're making. However, we do respect your point and we are preparing an FAQ to more fully explain our licensing as well as how it relates to the OSI (specifically, being "source available" under the OSI definition of "open-source").

**Comment by reubano** [comment]
Thanks for the thoughtful reply @jlowin! In addition to the points above, a key reason to use a common license (OSI approved or not) is so that I (or anyone else) can easily determine what they are (dis)allowed to do without having a lawyer review the license. Also, it makes figuring out license compatibility much easier. 

FYI, relevant [Timescale DB post](https://blog.timescale.com/blog/building-open-source-business-in-cloud-era-v2/) and related [HN discussion](https://news.ycombinator.com/item?id=24579905).
